scarb/src/bin/scarb/args.rs

```diff
@@ -268,6 +268,12 @@ pub struct ScriptsRunnerArgs {
     pub args: Vec<OsString>,
 }
 
+#[derive(ValueEnum, Clone, Debug)]
+pub enum TestRunner {
+    StarknetFoundry,
+    CairoNativeRunner,
```

Comment:
```suggestion
    CairoTest,
```

I think that `cairo-test` is usually used to refer to that